ce3/shared/src/main/scala/org/typelevel/log4cats/ce3/IOLocalHelpers.scala

```diff
@@ -0,0 +1,16 @@
+package org.typelevel.log4cats.ce3
+
+import cats.effect.{IO, IOLocal}
+import org.typelevel.log4cats.{LoggerFactory, SelfAwareStructuredLogger}
+
+object IOLocalHelpers {
+  def loggerWithContextFromIOLocal(
+      sl: SelfAwareStructuredLogger[IO],
+      local: IOLocal[Map[String, String]]
+  ): SelfAwareStructuredLogger[IO] = SelfAwareStructuredLogger.withContextF(sl)(local.get)
+
+  def factoryWithContextFromKleisli(
```

Comment:
Should this be `FromIOLocal` rather than `FromKleisli`?

Reply:
Well spotted 👁
Thanks 🙏